src/main/java/fr/xephi/authme/api/v3/AuthMeApi.java

```diff
@@ -196,6 +196,32 @@ private Long getLastLoginMillis(String playerName) {
         return null;
     }
 
+    /**
+     * Get the registration (AuthMe) timestamp of a player.
+     *
+     * @param playerName The name of the player to process
+     *
+     * @return The timestamp of when the player was registered, or null if the player doesn't exist or is not registered
+     */
+    public Instant getRegistrationTime(String playerName) {
+        Long registrationDate = getRegistrationMillis(playerName);
+        return registrationDate == null ? null : Instant.ofEpochMilli(registrationDate);
+    }
+
+    private Long getRegistrationMillis(String playerName) {
+        if (!isRegistered(playerName.toLowerCase())) {
```

Comment:
I think that this check can be avoided as we already check if auth != null below

Reply:
Done